event-app-api/server.js

```diff
@@ -94,14 +136,61 @@ app.get("/users", async (req, res) => {
   }
 });
 
-sequelize
-  .sync({ alter: true })
-  .then(() => {
-    const port = 3000;
-    app.listen(port, () => {
-      console.log(`App is listening on port ${port}`);
+//sending a get request to the events endpoint
+app.get("/events", async (req, res) => {
+  try {
+    const events = await Event.findAll({
+      // making the order of the events in a descending order depending on the time created
+      order: [["createdAt", "DESC"]],
```

Comment:
It would make more sense for the order to be ascending event start time, not descending created time. Let's say I create an event for next Monday, and then I create the event for next Friday. This query would return the Friday one first, and then the Monday one, which is not as user-friendly. Also you should consider querying for only the events in the future.